cli/buildengine.ts

```diff
@@ -58,10 +58,10 @@ export const buildEngines: Map<BuildEngine> = {
 
     dockeryotta: {
         id: "dockeryotta",
-        updateEngineAsync: () => runDockerAsync(["yotta", "update"]),
-        buildAsync: () => runDockerAsync(["yotta", "build"]),
+        updateEngineAsync: () => runDockerYottaAsync(["yotta", "update"]),
+        buildAsync: () => runDockerYottaAsync(["yotta", "build"]),
```

Comment:
If we are special casing Yotta, do we need to include "yotta" in the args?

Reply:
I think it's easier to understand with yotta there since that's the command you would be running if you were doing all this manually.

Reply:
Makes sense!